app/views/comments/_comment_date.erb

```diff
@@ -1,6 +1,15 @@
 <span class="color-base-30 px-2" role="presentation">&bull;</span>
+
 <a href="<%= URL.comment(decorated_comment) %>" class="comment-date crayons-link crayons-link--secondary fs-s">
   <time datetime="<%= decorated_comment.published_timestamp %>">
     <%= decorated_comment.readable_publish_date %>
   </time>
+
+  <% if decorated_comment.edited_at.present? %>
+    • Edited
```

Comment:
```suggestion
    &bull; Edited
```
Let's use the html entity for this (which matches what we do on line 1!):
```html
<span class="color-base-30 px-2" role="presentation">&bull;</span>
```

Reply:
thanks!